tests/Unit/Rules/CnesXMLIdentificationTest.php

```diff
@@ -16,21 +16,45 @@ class CnesXMLIdentificationTest extends TestCase
 	public function testPasses()
 	{
 		$ibge_code = '12345';
+
 		$this->mockXmlContents(['ibge_code' => $ibge_code]);
 
-		$rule = new CnesXMLIdentification($ibge_code);
+		$rule = new CnesXMLIdentification($ibge_code, '2.1');
 
 		$this->assertTrue($rule->passes('file', UploadedFile::fake()->create('xml.zip')));
 	}
 
+	public function testNotPassesVersionXSD()
+	{
+		$ibge_code = '12345';
+
+		$this->mockXmlContents(['ibge_code' => $ibge_code, 'version_xsd' => '']);
+
+		$rule = new CnesXMLIdentification($ibge_code, '2.1');
+
+		$this->assertFalse($rule->passes('file', UploadedFile::fake()->create('xml.zip')));
+	}
+
+	public function testNotPassesVersionXSDDiff2_1()
+	{
+		$ibge_code = '12345';
+		$version_xsd = '3.0';
+
+		$this->mockXmlContents(['ibge_code' => $ibge_code, 'version_xsd' => "VERSION_XSD=\"$version_xsd\""]);
```

Comment:
Linha maior que 100 caracteres, talvez seria interessante seguir o padrão do outro valor do array (ibge_code) que é apenas o valor no xml,  desta forma o `VERSION_XSD=` ficaria na string que corresponde ao xml e no array só passa o valor do version_xsd

Reply:
entendi, então basta corrigir a linha maior que 100 caracteres kk

Reply:
blz.. eu to um pouco corrido, mas pretendo fazer isso até o final da semana